content/en/news/releases/1.22.x/announcing-1.22/change-notes/index.md

```diff
@@ -0,0 +1,246 @@
+
+- **Enabled** configuring Dynatrace Sampler for the `OpenTelemetryTracingProvider` in `MeshConfig`.
+  ([Issue #50001](https://github.com/istio/istio/issues/50001))
+
+- **Enabled** configuring Resource Detectors for the `OpenTelemetryTracingProvider` in `MeshConfig`.
+  ([Issue #48885](https://github.com/istio/istio/issues/48885))
+
+- **Fixed** an issue that `TraceId` is not propagated when using OpenTelemetry access logger.
+  ([Issue #49911](https://github.com/istio/istio/issues/49911))
+
+- **Removed** default tracing configuration that enables tracing to `zipkin.istio-system.svc`. See upgrade notes for more information.
+
+## Extensibility
+
+- **Improved** Use tag-stripped URL + checksum as a Wasm module cache key, and the tagged URL is separately cached.
+This may increase the chance of cache hit (e.g., trying to find the same image with both of the tagged and digest URLs.)
+In addition, this will be a base to implement `ImagePullPolicy`.
+
+## Installation
+
+- **Improved** helm value field names to configure whether an existing CNI install
+ will be used. Instead of `values.istio_cni` the enablement fields will be in
+ `values.pilot.cni` as istiod is the affected component.
+ That is clearer than having `values.cni` for install config and `values.istio_cni`
+ for enablement in istiod. The old `values.istio_cni` fields will still be supported
+ for at least two releases.
+  ([Issue #49290](https://github.com/istio/istio/issues/49290))
+
+- **Improved** the `meshConfig.defaultConfig.proxyMetadata` field to do a deep merge when overridden, rather than replacing all values.
+
+- **Added** Allow user to add customized annotation to istiod service account resource through helm chart.
+
+- **Added** `openshift-ambient` profile.
+  ([Issue #42341](https://github.com/istio/istio/issues/42341))
+
+- **Added** a new, optional experimental admission policy that only allows stable features/fields to be used in Istio APIs
+  ([Issue #173](https://github.com/istio/enhancements/issues/173))
+
+- **Added** support for configuring ca bundle for validation and injection webhook
+
+- **Fixed** Gathering `pprof` data from the local ztunnel admin endpoint would fail due to lack of writable in-container `/tmp`
+  ([Issue #50060](https://github.com/istio/istio/issues/50060))
+
+- **Removed** deprecated `external` profile, use `remote` profile instead for installation.
+  ([Issue #48634](https://github.com/istio/istio/issues/48634))
+
+## istioctl
+
+- **Updated** `istioctl experimental proxy-stauts` has been promoted to `istioctl proxy-status`. The old `istioctl proxy-status` command has been removed.
+This promotion should not result in any loss of functionality. However, the request is now sent based on xDS instead of HTTP, and we have introduced a set of new xDS-based flags to target the control plane.
```

Comment:
```suggestion
- **Added** the `istioctl proxy-stauts` command, which is the promoted `istioctl experimental proxy-status` command. The old `istioctl proxy-status` command has been removed.
  This promotion should not result in any loss of functionality. However, the request is now sent based on xDS instead of HTTP, and we have introduced a set of new xDS-based flags to target the control plane.
```